api/python/cellxgene_census/src/cellxgene_census/experimental/ml/pytorch.py

```diff
@@ -444,47 +484,46 @@ def _init(self) -> None:
         self._initialized = True
 
     @staticmethod
-    def _partition_obs_joinids(ids: npt.NDArray[np.int64]) -> npt.NDArray[np.int64]:
-        dist_partition = loader_partition = 0
-        dist_partitions = loader_partitions = 1
+    def _subset_ids_to_partition(
```

Comment:
The new `_subset_ids_to_partition()` and `_compute_partitions()` methods are introduced only as a refactoring to accommodate the explicit list of (possibly) shuffled obs_joinids.  The partitioning logic is effectively the same as before, except that it now operates at the chunk level rather than the row level.